torch/distributed/distributed_c10d.py

```diff
@@ -1020,7 +1067,7 @@ def send(tensor, dst, group=None, tag=0):
         group.send([tensor], group_dst_rank, tag).wait()
 
 
-def recv(tensor, src=None, group=None, tag=0):
+def recv(tensor: torch.Tensor, src: Optional[int] = None, group: Optional[ProcessGroup] = None, tag: int = 0) -> Work:
```

Comment:
Wonder if there's anything we can do to automatically lint that future distributed PRs introducing a new public API must be typed.